AnkiConnect.py

```diff
@@ -434,6 +434,62 @@ def guiAddCards(self):
         addCards.activateWindow()
 
 
+    def guiGetNextCard(self):
+        if self.window().reviewer.card is None or self.window().state != 'review':
+            self.window().moveToState('review')
+
+        card = self.window().reviewer.card
+        return {
+            'success': True,
+            'id': card.id,
+            'question': card._getQA()['q'],
+            'answer': card._getQA()['a'],
+            'answerButtons': self.window().reviewer._answerButtonList(),
+            'modelName': card.note(reload)._model['name'],
+            'ord': card.ord,
+            'fields': card.note(reload).fields,
+            'fieldMap': card.note(reload)._fmap
+        }
+
+
+    def guiShowQuestion(self):
+        if self.window().reviewer.card is None or self.window().state != 'review':
+            self.window().moveToState('review')
+
+        self.window().reviewer._showQuestion()
+        return {'success': True}
+
+
+    def guiShowAnswer(self):
+        if self.window().reviewer.mw.state != 'review':
+            return {'success': False, 'message': 'Window state is not review.'}
+        else:
+            self.window().reviewer._showAnswer()
+            return {'success': True}
+
+
+    def guiAnswerCard(self, id, ease):
+        if self.window().reviewer.mw.state != 'review':
+            return {'success': False, 'message': 'Window state is not review.'}
```

Comment:
As I mentioned in the previous review, to keep things consistent with the rest of the API, we should not return success state with `success` and `message` variables. Just return `True` or `False` for actions or `None` when the caller expects a result.